test/test_unit.jl

```diff
@@ -1076,6 +1076,46 @@ end
     end
 end
 
+@timed_testset "Consistency check for HLLC flux: CEE" begin
```

Comment:
We also have some checks whether `FLuxRotated` with the Cartesian versions yields the curved implementation. Could you please test the new methods there, too?